msa/tb/docker-postgres/start-db.sh

```diff
@@ -22,14 +22,22 @@ PG_CTL=$(find /usr/lib/postgresql/ -name pg_ctl)
 if [ ! -d ${PGDATA} ]; then
     mkdir -p ${PGDATA}
     ${PG_CTL} initdb
+else
+    ${PG_CTL} start
 fi
 
 exec setsid nohup postgres >> ${PGLOG}/postgres.log 2>&1 &
 
 if [ ! -f ${firstlaunch} ]; then
     sleep 2
-    while ! psql -U ${pkg.user} -d postgres -c "CREATE DATABASE thingsboard"
+    while ! psql -U thingsboard -d postgres -c "CREATE DATABASE thingsboard"
     do
       sleep 1
     done
-fi
\ No newline at end of file
+else
+    until pg_isready --dbname thingsboard --quiet
```

Comment:
Infinite awaiting is quite an unusual solution. Maybe some retry count is a better option to await the Postgres.

Here some example how to avoid infinite loop:

```
RETRIES=30
until psql -U ${pkg.user} -d postgres -c "select 1" > /dev/null 2>&1 || [ $RETRIES -eq 0 ]; do
  echo "Connecting to Postgres , $((RETRIES--)) attempts left..."
  sleep 1
done
```